scripts/commands/checkLinks.ts

```diff
@@ -259,7 +303,7 @@ async function main() {
 
   // Validate external links
   // A for loop is used to reduce the risk of rate-limiting
-  if (args.external) {
+  if (args["external"]) {
```

Comment:
I think this might turn off type hinting? What does VSCode show the type as here?

Reply:
VSCode shows the type `(property) Arguments.external: boolean`